lib/ldap_fluff/posix_member_service.rb

```diff
@@ -16,14 +16,17 @@ def find_user(uid, base_dn = @base)
   # return an ldap user with groups attached
   # note : this method is not particularly fast for large ldap systems
   def find_user_groups(uid)
+    user = find_user(uid).first
```

Comment:
Should this somehow yell if there's more than 1 user?

Reply:
Can it happen? I was living under the impression that the uid we get should uniquely identify a single object within the subtree set by `@base`

Reply:
It wouldn't be a good practice, but if you organize your LDAP tree with different organization units it can happen.

Concrete example: let's say your base is `dc=example,dc=com` you can have `cn=myuid,ou=People,dc=example,dc=com` and `cn=myuid,ou=Systems,dc=example,dc=com`.

Reply:
There can be multiple entries with the same uid (as opposed to dn)